ContactDetailsApi/V2/Gateways/ContactDetailsDynamoDbGateway.cs

```diff
@@ -51,7 +51,7 @@ public async Task<List<ContactDetails>> GetContactDetailsByTargetId(ContactQuery
                 contactDetailsEntities.AddRange(newResults);
             } while (!search.IsDone);
 
-            return contactDetailsEntities.ToDomain();
+            return contactDetailsEntities?.ToDomain();
```

Comment:
What would this fix? I dont think contractDetailsEntity would ever be null.

Reply:
It is on staging - I deployed this change to staging and it resolves the issue

Reply:
Interesting